### ISSUE TITLE
Notice: TraceForwarder's queue is full, 200 log items were dropped! RTView shows idle

### ISSUE THREAD

**Issue by billcarroll** [input-output-hk/cardano-rt-view#162]
When Nodes are first started RTView works normally and web interface displays reported values as expected. 

Within a few minutes after nodes are started this message begins appearing in groups.

"Notice: TraceForwarder's queue is full, 200 log items were dropped!" 

In addition to messages on nodes, rtview shows nodes as idle with no data

Debian 10
node 1.25.1
rtview 0.3.0

**Comment by billcarroll** [comment]
When configuring RTView on run, the instructions say to  Find options -> mapBackends in config.json for the node and redirect required metrics to "TraceForwarderBK" 

I added "KatipBK" to the node config and that seems to be working. Is "TraceForwarderBK" incorrect?

**Comment by denisshevchenko** [comment]
> "Notice: TraceForwarder's queue is full, 200 log items were dropped!"

Yes, we still have this problem. Currently, I'm working on a new forwarding model in `cardano-node`. After it will be done, this "overflow queue" problem will be solved.
